src/main/cpp/src/from_json_to_raw_map.cu

```diff
@@ -720,4 +722,57 @@ std::unique_ptr<cudf::column> from_json_to_raw_map(cudf::strings_column_view con
                                  mr);
 }
 
+std::pair<rmm::device_buffer, char> concat_json(cudf::column_view const& input,
+                                                rmm::cuda_stream_view stream,
+                                                rmm::device_async_resource_ref mr)
+{
+  CUDF_FUNC_RANGE();
+  auto const input_scv = cudf::strings_column_view{input};
+
+  auto constexpr num_levels = 256;
+  rmm::device_uvector<uint32_t> d_histogram(num_levels, stream);
+  thrust::fill(rmm::exec_policy(stream), d_histogram.begin(), d_histogram.end(), 0);
+  auto lower_level = std::numeric_limits<char>::min();
+  auto upper_level = std::numeric_limits<char>::max();
+
+  size_t temp_storage_bytes = 0;
+  cub::DeviceHistogram::HistogramEven(nullptr,
+                                      temp_storage_bytes,
+                                      input_scv.chars_begin(stream),
+                                      d_histogram.begin(),
+                                      num_levels,
+                                      lower_level,
+                                      upper_level,
+                                      input_scv.chars_size(stream),
```

Comment:
`input_scv.chars_begin(stream)` and `input_scv.chars_size(stream)` are called here and below, each is called twice. So there are 4 stream syncs. We can do better by manually calling the underlying `memcpy` to ultimately have just 1 stream sync.

Reply:
Oh I was wrong. `chars_begin` doesn't have stream sync. So we only need to cache `chars_size` out of the function parameters.